setup.py

```diff
@@ -164,16 +248,42 @@ def runtest(self):
     license="BSD",
     long_description="",
     name="cellprofiler",
+    options = {
+        "py2exe": {
```

Comment:
I started to do this, then ran into the problem that the packages dictionary entry is computed rather than hardcoded. I did try adding an options section for "foo" to see if setup.py would run with options to a non-existent command and it did, so it's not a blocker and we can talk about hardcoding the packages list.